lib/qx/tool/cli/commands/contrib/Update.js

```diff
@@ -263,25 +264,25 @@ qx.Class.define("qx.tool.cli.commands.contrib.Update", {
               // qooxdoo version @deprecated
               var qx_versions = data.info["qooxdoo-versions"];
               // @deprecated
-              var qx_version_range = data.requires["qooxdoo-sdk"] || data.info["qooxdoo-range"];
+              var qx_version_range = (data.requires && data.requires.libraries && data.requires.libraries.qx) || data.info["qooxdoo-range"];
```

Comment:
Interesting! I did not know that in JavaScript `("blue" && "black" && "green")` will return `"green"` and not boolean  `true`....

Reply:
I learned this yesterday: 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_Operators